lib/paperclip/storage/s3.rb

```diff
@@ -124,6 +124,7 @@ module Storage
     module S3
       def self.extended(base)
         begin
+          gem "aws-sdk-s3", ">= 1.197.0"
```

Comment:
Ideally, this should introduce some conditional logic, depending on the gem version and support older and newer. But I think, it is just easier to require the minimum version.